### ISSUE TITLE
Can't export to PDF using Microsoft Edge browser in Windows 10

### ISSUE THREAD

**Issue by rlktradewright** [benweet/stackedit#940]
I signed up as a sponsor on 15 February 2015. But now I can't persuade StackEdit to allow me to export to PDF. There is a sign-in link in the 'Sponsor only!' dialog that pops up. This brings up the monetize website, where I can sign in (though I had to reset my password first since it seemed to have forgotten my original password). But even after signing in, I still can't export to PDF.

Sorry if I sound irritated (I also maintain an open source project so I understand the pressures), but this is incredibly annoying, as I have to embark on a journey in the next twenty minutes and I really needed to export this PDF before then. 

Guess I'm just going to have to wait till I get back in a week's time...

Richard King

**Comment by BigBlueHat** [comment]
Did you get an error? I was able to do it just now--had to download the PDF vs. opening it in Firefox, but otherwise it worked fine.

**Comment by rlktradewright** [comment]
I discovered that my problem was caused by using Microsoft Edge browser in Windows 10, which evidently isn't supported by StackEdit. Major fail in my view!

Using Chrome instead worked fine, but I detest Chrome (and Firefox) and I'd really rather not have them infesting my computers!

I've updated the title of this issue to reflect the situation, in the hope that someone might get this sorted at some point.

**Comment by BigBlueHat** [comment]
Well...given that Edge wasn't a thing when StackEdit was created...it's a new-ish problem. :stuck_out_tongue: 

StackEdit seems to work best in Chrome, and should probably (until the other bits get fixed) be considered as only working in Chrome--I'm a 15+ year Firefox user, so not keen on that either, but there are other things here I'd love to see folks help out with to get fixed.

@rlktradewright could you do some more debugging in Edge and post some error messages / screenshots? It'd certainly help others debug--especially since I'm confident most here are on Macs and can't use Edge.

**Comment by rlktradewright** [comment]
Oops, how embarrassing, my previous post is nonsense, and I apologise unreservedly for any confusion I've caused!

Firstly, the original problem occurred not on Edge on Windows 10, but on IE 11 on Windows 7. I've just tested on Edge and it works fine.

Second, the problem with IE11 is evidently fixed, as I just tested it there too with no problem. Perhaps this is not surprising since I reported it two months ago, but I assumed that nothing had changed because I hadn't had any feedback on this issue.

So now I'll crawl back to my corner and consider this issue closed!
